website/docs/docs/dbt-versions/release-notes/08-May-2023/deployment-monitor.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Use the Deployment monitor page for greater visibility to your job run health"
+id: "deployment-monitor"
+description: "May 2023: "
+sidebar_label: "New: Deployment monitor visualizes your run health"
+sidebar_position: 5
+tags: [scheduler]
+---
+
+The new Deployment monitor helps you gain greater visibility into your deployment health and quickly detect any issues with the improved filtering and sorting capabilities. You can now easily curate the view based on environment, prioritizing the ones that matter most, such as Production. With pre-filtered lists for in progress and in queue activities, it's quicker and easier to track the relevant activity and stay on top of your deployment status.
```

Comment:
```suggestion
The new **Deployment Monitor** page helps you gain greater visibility into your deployment health and quickly detect any issues with the improved filtering and sorting capabilities. You can now easily curate the view based on environment, prioritizing the ones that matter most, such as Production. With pre-filtered lists for in progress and in queue activities, it's quicker and easier to track the relevant activity and stay on top of your deployment status.
```

Reply:
oh, i see. you're right, **Production** is the correct font decoration if that was your intention. 

maybe for the sake of clarity, suggest clarifying the exact UI element you're referring to like "such as **Production** from .... " using location, UI component, or how to see it (get to it)

Reply:
good callout, made that tweak!